content/docs/2.5/concepts/scaling-jobs.md

```diff
@@ -193,6 +194,19 @@ if (maxScale + runningJobCount) > maxReplicaCount {
 ```
 For more details,  you can refer to [this PR](https://github.com/kedacore/keda/pull/1227).
 
+---
+
+```yaml
+scalingStrategy:
+    MultipleScalersOption: "max" # Optional. Default: max. If multiple scaler is exists, how to calculate the queueLength and maxValue
+```
+Select a behavior if you have multiple triggers. Possible values are `max`, `min`, `avg`, or `sum`. The default value is `max`. 
+
+* **max:** default, pick a scaler that has the max number of `queueLength`.
```

Comment:
What does queueLength refer to here? Is that a metric that I have defined or an internal one? Do end-users care about this metric or should it be "pending jobs"?

Reply:
Thank you, @tomkerkhove for the great review! 

queueLength is the metadata that is included on scaler.GetMetrics. It is the number of the queue that is not consumed. In case of "pending jobs" it might confusing since ScaledJob create jobs and some of them called pending jobs.  I'm wondering what is the proper expression in English. 

https://github.com/kedacore/keda/blob/main/pkg/scaling/scale_handler.go#L283
For the DCO issue that is created GitHub gui, I'll fix it. later. :)

Reply:
Hi @tomkerkhove If you don't have an idea, how about adding explanation of the `queueLength` ? e.g. "queueLength is the number of the queues that is not consumed. The value is returned from each scalers"

Reply:
It's ok for me